src/components/index.ts

```diff
@@ -67,7 +67,39 @@ export { ExpandingIconButton } from './Inputs/ExpandingIconButton/ExpandingIconB
 export type { ExpandingIconButtonProps } from './Inputs/ExpandingIconButton/ExpandingIconButton';
 export { default as FileUploadArea } from './Inputs/FileUploadArea';
 export { default as IconToggleButton } from './Inputs/IconToggleButton';
-export { default as RichTextEditor } from './Inputs/RichTextEditor/RichTextEditor';
+export {
+  default as RichTextEditor,
+  EditorStyling,
```

Comment:
Nitpicky, should we have all the compound components related to RichTextEditor be exported something like this? That way it becomes really obvious that these are RichTextEditor related.
```tsx
// Topbar example: 
import ApplicationDrawer from './ApplicationDrawer/ApplicationDrawer';
import { GuidelineSections } from './Guidelines/Guidelines';
import Notifications from './Notifications/Notifications';
import { Resources } from './Resources/Resources';
import { Actions } from './Actions';
import FieldSelector from './FieldSelector';
import Guidelines from './Guidelines';
import { ISettingsProps, Settings } from './Settings';
import { TopBar as BaseTopBar } from './TopBar';
import { Account } from 'src/components/Navigation/TopBar/Account/Account';

type TopBarType = typeof BaseTopBar & {
  Account: typeof Account;
  Actions: typeof Actions;
  Guidelines: typeof Guidelines;
  Settings: typeof Settings;
  Notifications: typeof Notifications;
  FieldSelector: typeof FieldSelector;
  Resources: typeof Resources;
  ApplicationDrawer: typeof ApplicationDrawer;
};

const TopBar = BaseTopBar as TopBarType;
TopBar.Account = Account;
TopBar.Actions = Actions;
TopBar.Guidelines = Guidelines;
TopBar.Settings = Settings;
TopBar.Notifications = Notifications;
TopBar.FieldSelector = FieldSelector;
TopBar.Resources = Resources;
TopBar.ApplicationDrawer = ApplicationDrawer;

export default TopBar;
```

Reply:
Done 😊 I now export them like this:

```JS
const RichText = {
  Styling: EditorStyling,
  Provider: EditorProvider,
  Content: EditorContent,
  Bar: AmplifyBar,
  Kit: AmplifyKit,
};

export type { RichTextEditorProps, OnImageUploadFn, RichTextEditorFeatures };
export { RichText };
export default RichTextEditor;

```

That way you can import like this: 

```JS
import RichTextEditor, {
  RichText,
  RichTextEditorProps,
} from './RichTextEditor';
```

Which lets us use the RichText key as a buss for the compound components while keeping the higher level RichTextEditor default export untouched so that we dont break peoples imports 😊

Reply:
Actually, we could consider doing this modification: 

```JS
const RichText = {
  Styling: EditorStyling,
  Provider: EditorProvider,
  Content: EditorContent,
};

const Amplify = {
  Bar: AmplifyBar,
  Kit: AmplifyKit,
};
```

Idk, is it better to keep them together or separate like this? Not sure theres a right or wrong answer. Your call.

Reply:
I like having them all in one like you already did 😄

Reply:
Cool. Lets do it like that then